### DISCUSSION TITLE
Recommended input size for models with EfficientNetV2 backbones

### DISCUSSION THREAD

**Discussion by arekmula** [rwightman/efficientdet-pytorch#259] [Q&A]
Hi,

What is the recommended input size for object detection models that are using EfficientNetv2 backbones? 

In this [tutorial](https://gist.github.com/Chris-hughes10/73628b1d8d6fc7d359b3dcbbbb8869d7) he's using 512x512 input for the `tf_efficientnetv2_l` backbone, while the input_size for this backbone is 384x384. 

EfficientDet v1 has precise information about input size of the images:
![image](https://user-images.githubusercontent.com/61802388/140721213-babcca39-a703-4dbb-95ad-35c88ce87a07.png)

**Comment by rwightman** [comment]
@arekmula I'm currently a bit over 1/4 through training a v2_tiny `efficientnetv2_rw_t` (my variant that's roughly B3 level for classification) with roughly D2 detection config, slightly more channels (128) but same res and bifpn layers.

It's ultimately up to you  in terms of tradeoffs, but using similar scaling to the original would put most of the v2 models into pretty expensive territory (in terms of compute / time). Scaling the resolution really blows up your costs. Hence why I used my smallest v2 w/ a step down from equivalent detection config.

**Reply by rwightman** [reply]
For next trial , if/when I get there I'd likely pair a V2-small w/ a slightly larger than D3ish config

**Reply by arekmula** [reply]
Thank you for the advice :)
